packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.service.js

```diff
@@ -99,6 +100,25 @@ export default class IpAgoraOrder {
       .catch(() => false);
   }
 
+  getIpFailoverRIPEPrice(ovhSubsidiary = 'FR') {
+    return this.$http
+      .get(`/order/catalog/formatted/ip?ovhSubsidiary=${ovhSubsidiary}`)
+      .then(({ data: { plans } }) => {
```

Comment:
```suggestion
      .then(({ data: { plans = [] } }) => {
```

Reply:
Not sure about this change
If `plans` is not an array, it means that something went wrong, we should let it be caught